# backup.tst

    >>> from Test.backup import driver
    >>> driver.run()
    Traceback (most recent call last):
        ...
    KeyError: '$ans not bound'

